exercises/07-normalizr/src/modules/entities/entities-schema.ts

```diff
@@ -0,0 +1,22 @@
+import { schema } from 'normalizr'
+
+export const skill = new schema.Entity('skills')
+export const skills = [skill]
+
+export const usersSkill = new schema.Entity(
```

Comment:
Thumbs up for this. A lot of people realise this concept sometimes too late :D

Reply:
which concept? 👀